src/javascript/index.ts

```diff
@@ -10,3 +10,5 @@ export * from "./projenrc";
 export * from "./typescript-config";
 export * from "./upgrade-dependencies";
 export * from "./yarnrc";
+export * from "./biome";
+export * from "./biome-config";
```

Comment:
There's a lot of names in this generated file. We should namespace this if possible:

```suggestion
export * as BiomeConfig from "./biome-config";
```

Disclaimer: I don't know if that works with jsii.

Reply:
Namespaces seem to be supported (had to check also self with build), but just a bit different format:
```
src/javascript/index.ts:13:13 - error JSII8004: Submodule namespaces must be camelCased or snake_cased. Rename "Biome" to biome

13 export * as Biome from "./biome/biome";
               ~~~~~
src/javascript/index.ts:14:13 - error JSII8004: Submodule namespaces must be camelCased or snake_cased. Rename "BiomeConfig" to biomeConfig

14 export * as BiomeConfig from "./biome/biome-config";
```